### DISCUSSION TITLE
Final submission page down

### DISCUSSION THREAD

**Discussion by Dru-Mara** [snap-stanford/ogb#224] [General]
Hello,

I'm not sure if anyone else has noticed this, but it seems that the submission page (https://ogb-save.stanford.edu/final/) is unreachable. I've been trying to access it for several hours now without any success. Can you guys take a look? 

Many thanks.

**Comment by Dru-Mara** [comment]
Seems to be solved now!

**Reply by weihua916** [reply]
Great, thank you for letting us know!
